app/src/index.py

```diff
@@ -11,27 +11,17 @@
 
 
 navbar = dbc.Navbar(
-    [
-        dbc.Row(
-            [
-                dbc.Col(dbc.NavbarBrand("Outil d'aide à la création de BEGES", className="ml-2"))
-            ]
-        )
-    ],
-    className="nav_bar"
+    [dbc.Row([dbc.Col(dbc.NavbarBrand("Outil d'aide à la création de BEGES", className="ml-2"))])], className="nav_bar"
 )
 
 app.layout = dbc.Container(
     [
         html.Div(id="div-url-redirect"),
         dcc.Location(id="url", refresh=False),
         navbar,
-        #html.H1("Outil d'aide à la création de BEGES", style={"text-align": "center", "margin-top": "5%"}),
-        #html.Hr(),
         html.Br(),
         html.Div(id="page-content"),
-    ],
-    fluid=True
```

Comment:
I don't know how to fix the layout to take 100% of the display without setting the container to fluid. Maybe we should keep it for now. Any thoughts ?

Reply:
Fixed.